packages/mgt-components/src/components/mgt-file/mgt-file.ts

```diff
@@ -38,7 +38,8 @@ import { strings } from './strings';
  * @class MgtFile
  * @extends {MgtTemplatedComponent}
  *
- * @cssprop --file-type-icon-size - {Length} file type icon size
+ * @cssprop --file-type-icon-width - {Length} file type icon width
```

Comment:
I'm not seeing any usage of `--file-type-icon-width` in the `mgt-file.scss` should this prop be here, or was it overlooked in the scss file?

Reply:
should be --file-type-icon-height